prow/repoowners/repoowners_test.go

```diff
@@ -512,7 +799,12 @@ func TestLoadRepoOwners(t *testing.T) {
 			continue
 		}
 		ro := r.(*RepoOwners)
-
+		if test.expectedReusable {
```

Comment:
Please also validate the negative case. i.e. If we don't expect the cache entry to be reused the baseDir should not be "cache".

Reply:
Fixed